integrations/pinterest-tag/lib/index.js

```diff
@@ -132,6 +133,10 @@ Pinterest.prototype.createPropertyMapping = function() {
     quantity: 'product_quantity',
     brand: 'product_brand'
   };
+
+  if(this.options.mapMessageIdToEvnentId){
+    this.productPropertyMap.messageId = 'event_id';
```

Comment:
Why is messageId <-> even_id mapping being added in `productPropertyMap` and not in `propertyMap`? It seems like propertyMap is used for looking up fields in [properties.product](https://github.com/segmentio/analytics.js-integrations/blob/3cf50f1b7fa971d519b227d54f87dbf21d23bb2a/integrations/pinterest-tag/lib/index.js#L157) path and not at the root level of the event where `messageId` is present.